packages/vx-tooltip/Readme.md

```diff
@@ -4,10 +4,127 @@
 npm install --save @vx/tooltip
 ```
 
-The `@vx/tooltip` package provides utilities for making it easy to add `Tooltip`s to a visualization and includes higher-order component (HOC) enhancers and Tooltip components.
+The `@vx/tooltip` package provides utilities for making it easy to add `Tooltip`s to a visualization and includes hooks, higher-order component (HOC) enhancers and Tooltip components.
 
-### Example:
-``` js
+### Hooks and Enhancers
+
+This package provides two ways to add tooltip state logic to your chart components:
+
+- a hook: `useTooltip()` 
+- a higher order component (HOC): `withTooltip()`
```

Comment:
If `withTooltip` is to be removed (Or deprecated`), docs about HOC should probably be removed, too.

Reply:
I don't think there's any reason to deprecate the HOC, I think this would be an unnecessary breaking change. 

They are complementary approaches and there are valid uses for both for now.